domains/ukrio.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Self hosted website, and an app.",
+
+  "domain": "is-cool.dev",
+  "subdomain": "ukrio",
+
+  "owner": {
+    "email": "opendomains@uk.is-a.dev"
+  },
+
+  "record": {
+    "NS": ["love.ns.cloudflare.com", "pete.ns.cloudflare.com"]
```

Comment:
Provide a valid reason as to why you need NS records.

Reply:
I would like a way to manage the records with another service, and have cloudflare analytics. Also I need to use the rules customization cloudflare provides for the pending webapp.

Reply:
Sounds good, if you are found to violate these conditions, we have the right to terminate your domain.